db/init_db.js

```diff
@@ -1,21 +1,71 @@
 // code to build and initialize DB goes here
 const {
   client
-  // other db methods 
+  // createUser, createProducts, etc. 
 } = require('./index');
 
 async function buildTables() {
   try {
     client.connect();
+      await client.query(`
+        DROP TABLE IF EXISTS users, products, orders, order_products;
+      `);
+  
+      await client.query(`
+      CREATE TABLE users(
+        id SERIAL PRIMARY KEY, 
+        firstname VARCHAR(255) NOT NULL, 
+        lastname VARCHAR(255) NOT NULL, 
+        email VARCHAR(255) UNIQUE NOT NULL, 
+        "imgURL" DEFAULT VALUE, 
+        username VARCHAR(255) NOT NULL, 
+        password VARCHAR(255) NOT NULL, 
+        "isAdmin" BOOLEAN DEFAULT false,
+        address NOT NULL
+      );
+    `);
+    await client.query(`
+    CREATE TABLE products(
+      id SERIAL PRIMARY KEY, 
+      name VARCHAR(255) UNIQUE NOT NULL, 
+      description STRING, 
+      price INTEGER NOT NULL, 
+      "imgURL" DEFAULT VALUE,  
```

Comment:
Update on DEFAULT constraint. The word value should be changed to what we actually want as the default value (e.g. if we want some default avatar).